src/stores/rootStore.ts

```diff
@@ -435,15 +384,15 @@ export const useRootStore = defineStore('root', {
       this.stateFiling = stateFilingResponse
     },
 
+    setNoRedirect (val: boolean) {
+      this.noRedirect = val
+    },
```

Comment:
I think you can delete this code since there is no longer a dashboard to display here. This was to prevent redirection to the Business Dashboard UI.